Ambit/Source/Ambit/Mode/ConfigImportExport.spec.cpp

```diff
@@ -1017,4 +1017,133 @@ void ConfigImportExportSpec::Define()
             JsonContent = nullptr;
         });
     });
+
+    Describe("OnExportMap()", [this]()
+    {
+        BeforeEach([this]()
+        {
+            // Generate World
+            World = FAutomationEditorCommonUtils::CreateNewMap();
+
+            // Create Mode Settings
+            GLevelEditorModeTools().ActivateMode(FAmbitMode::EM_AmbitModeId);
+
+            FAmbitMode* AmbitMode = FAmbitMode::GetEditorMode();
+            check(AmbitMode);
+
+            // Generate Exporter
+            Exporter = NewObject<UMockableConfigImportExport>();
+
+            auto MockListBuckets = []() -> TSet<FString>
+            {
+                TSet<FString> setBuckets;
+                setBuckets.Add("BucketName");
+                return setBuckets;
+            };
+            Exporter->SetMockS3ListBuckets(MockListBuckets);
+
+            auto MockCreateBucket = [](const FString& Region, const FString& BucketName) -> void
+            {
+                return;
+            };
+            Exporter->SetMockS3CreateBucket(MockCreateBucket);
+
+            auto MockS3FileUpload = [](const FString& Region, const FString& BucketName, const FString& ObjectName, const FString& FilePath) -> bool
+            {
+                return true;
+            };
+            Exporter->SetMockS3FileUpload(MockS3FileUpload);
+        });
+
+        It("Should Return FReply::Handled", [this]()
+        {
+            const FReply Reply = Exporter->OnExportMap();
+
+            TestTrue("Reply should be true on return", Reply.IsEventHandled());
+        });
+
+        AfterEach([this]()
+        {
+            GLevelEditorModeTools().DeactivateMode(FAmbitMode::EM_AmbitModeId);
+            Exporter = nullptr;
+            JsonContent = "";
```

Comment:
Minor: Unnecessary to empty out JsonContent if unused.

Reply:
removed